content/hi/docs/contribute/localization_hi.md

```diff
@@ -0,0 +1,162 @@
+
+* ### कार्यात्मकता महत्वपूर्ण है, भाषाई गौरव नहीं।
+  हाँ, हिंदी एक शानदार भाषा है जिसका विशाल इतिहास है, और अपनी भाषाई परंपरा पर गर्व करना ठीक है।
+
+  लेकिन, एक सॉफ़्टवेयर को उद्देश्य के लिए बनाया जाता है और जो कुछ इसकी क्षमता और उपयोग में बाधा डालता है, वह व्यर्थ होगा। केवल इसलिए कि हिंदी में एक शब्द का अनुवाद मौजूद है, इसका यह मतलब नहीं है कि हमें उसे उपयोग करना चाहिए। ```कुबेरनेट्स``` को क्षमता-नियंत्रण पदार्थ में अनुवाद करना शायद हिंदी के प्रति अधिक नुकसानदायक होगा क्योंकि इससे उपयोगकर्ताओं को भ्रमित किया जाएगा और कुबेरनेट्स को हिंदी समझने से रोकेगा। आम तौर पर आप किस बात को प्रयोग करना सही समझेंगे? _"एक क्षमता-नियंत्रण पदार्थ और मूषक-यंत्र।" या "एक कीबोर्ड और माउस।"?_
+
+  जो कुछ एक शैक्षणिक या रचनात्मक संदर्भ में काम करता है, वह सॉफ़्टवेयर में काम नहीं करेगा जो वास्तविकता में सभी प्रकार के शैक्षिक, सामाजिक और सांस्कृतिक पृष्ठभूमि वाले लोगों द्वारा उपयोग किया जाने की उम्मीद की जाती है। हमेशा विचार करें कि दैनिक संवादात्मक स्थिति में क्या उपयोग होने की संभावना है, और क्या एक साधारण बातचीत की स्थिति में शामिल होना चाहिए, जबकि किताब के पाठ्यक्रम या आधिकारिक सर्कुलर/नोटिस में क्या शामिल होना चाहिए।
+
+## हिंदी अनुवाद के दौरान, सरल और सुलभ शब्दों और वाक्यांशों का उपयोग करें।
+---
+कुबेरनेट्स डॉक्स को अनावश्यक रूप से औपचारिक लहजे से बचना चाहिए। यहां कुछ उदाहरण दिए गए हैं बचने के लिए शब्द और वाक्यांश
+
+| en-US source    | hi-IN word to avoid    | hi-IN word/phrase     |
+|-----------------|------------------------|-----------------------|
+| Select/choose/pick | चयन करें              | चुनें                     |
+| Furthermore        | इसकेअतिरिक्त            | साथ ही                  |
+| Therefore          | इसलिए                | इस कारण                |
+| Refer to           | संदर्भ करें              | देखें                     |
+| Change             | बदलें                 | परिवर्तन करें               |
+| Deselect           | इसका चयन रद्द करे       | अचयनित करे              |
+
+
+## अंग्रेज़ी शब्दों को हिंदी में लिखने के लिए एक मार्गदर्शिका
+---
+* व्याकरणिक नियम के अनुसार, विशेष शब्दों या अक्षरों को अंग्रेज़ी में लिखें और उनके बाद हिंदी अनुवाद को ब्रैकेट में दें। यह एक साधारण तथा स्वीकार्य तरीका है। उदाहरण के रूप में _PC_ को _पीसी (PC)_ लिखें।
+* इंग्लिश शब्दों की हिंदी में लिखने में संक्षेप (एब्रिविएशन) का प्रयोग करें, जब यह प्रचलित हो और जब शब्द का अनुवाद साफ और समझने में आसान हो। उदाहरण के रूप में, _आइटी (IT)_ के लिए _Information Technology_ का उपयोग करें।
+* समय-समय पर, कई इंग्लिश शब्दों का प्रयोग ऐसे ही हिंदी में किया जाता है, जैसे _कॉपी (Copy)_, _पेस्ट (Paste)_, _एक्सिट (Exit)_ आदि। इन शब्दों का उपयोग करना भी स्वीकार्य है।
+  
+#### यहां एक उदाहरण सारणी दी गई है।
+
```

Comment:
That may be applicable for some places, but at same time you can not apply these rule to many places. Like kubectl example - 

> kubectl get pods
kubectl get nodes

Reply:
I wrote this guide keeping in mind the general doc language that are meant to be translated , Indeed we cannot do anything with the cammands , so how about we add something for the commands like how we should not translate the kubectl commands like ```kubectl get pods```